### PR TITLE
add Chips and export fontCaps mixins

### PR DESCRIPTION
**Pull request by tihuan**
1. Add chip (or tag as called in our design system)
2. Export `fontCaps` mixins
3. Add Tooltip styling arrow example 

Screen Shot 2021-05-10 at 10 08 46 AM
Screen Shot 2021-05-10 at 10 08 50 AM
Screen Shot 2021-05-10 at 10 08 53 AM
Screen Shot 2021-05-10 at 10 08 55 AM
Screen Shot 2021-05-10 at 12 50 21 PM
Screen Shot 2021-05-10 at 12 50 24 PM

demo

### CODE REVIEW

File: src/core/Tooltip/index.stories.tsx [hunk 1]
```diff
@@ -32,3 +33,13 @@ export const Info = Template.bind({});
 Info.args = {
   inverted: false,
 };
+
+export const StyledArrow = Template.bind({});
+
+const arrow = css`
```
Comment (tihuan) [inline src/core/Tooltip/index.stories.tsx:39]: this demonstrates that consumer can add extra styles to arrow as needed

File: src/core/styles/common/mixins/fonts.ts [hunk 2]
```diff
@@ -26,6 +26,29 @@ export const fontBodyXs = fontBody("xs");
...
+    } = typography;
+
+    return css`
+      ${themeToCss(caps[fontSize])}
+      text-transform: uppercase;
+    `;
+  };
+};
+
+export const fontCapsXxs = fontCaps("xxs");
```
Comment (tihuan) [inline src/core/styles/common/mixins/fonts.ts:48]: Chip uses this mixin, so might as well export them

**Review by tihuan — COMMENTED** [review]
PTAL thank you!!

**Review by jshoe — APPROVED** [review]
Haha I like the name chips!

File: src/core/styles/common/mixins/fonts.ts (hunk 2 above)
Comment (j-x-han) [reply src/core/styles/common/mixins/fonts.ts:48]: Nice! I end up using this for `TooltipTable` as well, so thanks for adding! 😄

Comment (tihuan) [reply src/core/styles/common/mixins/fonts.ts:48]: Amazing!! Go team 🚀 !!

**Pull request merged**